ext/standard/basic_functions.c

```diff
@@ -379,12 +379,12 @@ ZEND_BEGIN_ARG_INFO_EX(arginfo_array_slice, 0, 0, 2)
 	ZEND_ARG_INFO(0, preserve_keys)
 ZEND_END_ARG_INFO()
 
-ZEND_BEGIN_ARG_INFO_EX(arginfo_array_merge, 0, 0, 1)
+ZEND_BEGIN_ARG_INFO_EX(arginfo_array_merge, 0, 0, 0)
 	ZEND_ARG_INFO(0, arr1) /* ARRAY_INFO(0, arg, 0) */
```

Comment:
I think we should drop this arg now.

Reply:
I assume you are referring to this line:

```c
ZEND_ARG_INFO(0, arr1)
```

I removed it, leaving only ZEND_ARG_VARIADIC_INFO. Same for array_merge_recursive.